package.json

```diff
@@ -110,10 +110,11 @@
   "scripts": {
     "start": "cross-env PORT=3001 craco start",
     "start:test": "cross-env PORT=3001 REACT_APP_TEST_MODE=true craco start",
-    "start:for-real-backend": "cross-env REACT_APP_BACKEND=http://localhost:3000 yarn start",
+    "start:for-real-backend": "cross-env PORT=3001 REACT_APP_BACKEND_BASE_URL=http://localhost:3000 craco start",
     "serve:build": "http-server build/ -s -p 3001 -P \"http://localhost:3001?\"",
-    "build:test": "cross-env REACT_APP_TEST_MODE=true craco build",
-    "build:production": "craco build",
+    "build:test": "cross-env REACT_APP_BACKEND_BASE_URL=/mock-backend REACT_APP_TEST_MODE=true craco build",
+    "build:mock": "cross-env REACT_APP_BACKEND_BASE_URL=/mock-backend craco build",
+    "build:production": "[[ -v REACT_APP_BACKEND_BASE_URL ]] && (cross-env craco build) || (echo -e '\n==============\nREACT_APP_BACKEND_BASE_URL not set.\nUse this task only if you want to create a production build with a real backend. Otherwise use build:mock\n==============\n'; exit 1)",
     "analyze": "cross-env ANALYZE=true craco build",
```

Comment:
This needs to be changed to one of the build variants.

Reply:
I changed this to mock as this is the only mode where we need the analytics now.